README.md

```diff
@@ -57,6 +70,16 @@ docker-compose up -d
   - phpmyadmin (should connect to mysql via host host.docker.internal)
   - We include MySQL in docker containers but because we believe database is important and you may lose you db once docker failed. Using a database server on local machine is out proposal: use `host.docker.internal` (for mac), `10.0.2.2` (for docker machine) for the hostname to connect to your main machine.
 
+- If you have ERROR that cannot start service nginx_main. Because apache2 (or other service , you can find) use same port with it 
+You must stop that service by
+```sh
+sudo service apache2 stop
+```
+And rerun Docker Compose.
+
+## How to verify things working
+  https://docs.google.com/document/d/1XOnTa_-c37IWCLhh-JJaE-7GpW5FEkI_a72tsluBJak/edit?usp=sharing
```

Comment:
You need to create a markdown file in `docs` folder and create a like to that file not a like you Gdocs one